rest-server/src/config/job.js

```diff
@@ -54,6 +54,31 @@ const jobConfigSchema = Joi.object().keys({
         gpuNumber: Joi.number()
           .integer()
           .default(0),
+        portList: Joi.array()
+          .items(Joi.object().keys({
+            label: Joi.string()
+              .regex(/^[A-Za-z0-9\-._~]+$/)
+              .required(),
+            startAt: Joi.number()
+              .integer()
+              .default(0),
+            portNumber: Joi.number()
+              .integer()
+              .default(1),
+          }))
+          .optional()
+          .default([
+            {
+              label: 'http',
+              startAt: 0,
```

Comment:
80?

Reply:
startAt 0 means any ports, the Launcher will select one.

---
In reply to: [176960001](https://github.com/Microsoft/pai/pull/409#discussion_r176960001) [](ancestors = 176960001)